Formula/pup.rb

```diff
@@ -17,25 +15,14 @@ class Pup < Formula
   end
 
   depends_on "go" => :build
-
-  go_resource "github.com/mitchellh/iochan" do
-    url "https://github.com/mitchellh/iochan.git",
-        :revision => "87b45ffd0e9581375c491fef3d32130bb15c5bd7"
-  end
-
-  go_resource "github.com/mitchellh/gox" do
-    url "https://github.com/mitchellh/gox.git",
-        :revision => "6e9ee79eab7bb1b84155379b3f94ff9a87b344e4"
-  end
+  depends_on "gox" => :build
 
   def install
     ENV["GOPATH"] = buildpath
     dir = buildpath/"src/github.com/ericchiang/pup"
     dir.install buildpath.children
 
-    Language::Go.stage_deps resources, buildpath/"src"
-    ENV.prepend_create_path "PATH", buildpath/"bin"
-    cd("src/github.com/mitchellh/gox") { system "go", "install" }
+    (buildpath/"bin").mkpath
```

Comment:
I don't think this one is needed

Reply:
Sorry, got this mixed up with the others. Fixed.